src/actions/definitionActions.js

```diff
@@ -72,3 +82,22 @@ export function revertDefinitionAction(definition, values, name) {
     return dispatch(actions.success({ componentsWithoutChanges }))
   }
 }
+
+export function browseDefinitionsAction(token, entity, name) {
+  return dispatch => {
+    const actions = asyncActions(name)
+    dispatch(actions.start())
+    return getLowScoreDefinitions(token, entity).then(
```

Comment:
this is ok for now but in the end we'll want to pass a filter/sort set to guide the query. it is not just low score. We will eventually also have to do pages etc. Perhaps just generalize the naming and we can add the additional filtering/sorting capabilities later